packages/ChangesReloaded-Core.package/CRBrowser.class/instance/getChangeListItemsFrom..st

```diff
@@ -0,0 +1,12 @@
+initializing
+getChangeListItemsFrom: anOrderedCollectionOfChangeRecords
+
+	| changeListItems |
+	changeListItems := SortedCollection new.
+	(anOrderedCollectionOfChangeRecords 
+		select: [:each | each isMethod or: each isClassComment])
+		do: [:each |
+					| changeListItem |
+					changeListItem := CRChangeListItem new changeRecord: each.
+					changeListItems add: changeListItem].
+	^ changeListItems asSortedCollection sort: [:a :b | a > b]
```

Comment:
I believe this is a relict from before, because now, we don't even need the temporary variable. This would be much cleaner:
```suggestion
	^ (anOrderedCollectionOfChangeRecords 
		select: [:each | each isMethod or: each isClassComment])
		collect: [:each | CRChangeListItem new changeRecord: each; yourself]
		reversed
```
Note that using `reversed` instead of sorting it also automatically fixes #109.

Reply:
You are right about `collect:`, we will implement that. About `reversed`: We did that on purpose to avoid a future (uncatched) bug, that someone gives in anOrderedCollectionOfChangeRecords that have no proper sorting at all and expects them to be sorted correctly afterwards

But maybe we should just add a test for that and use `reversed`